### DISCUSSION TITLE
The image is loading with a delay

### DISCUSSION THREAD

**Discussion by AlSajid** [vercel/next.js#59953] [Help]
### Summary

Need urgent help.

I am developing an e-commerce site. On the product details page, the product image is taking a significant time to be seen on the screen. This issue only happens on a first-time visit.

I have tried priority='true' and loading='earger'. But still not fixed.

Sample page: https://fejmo.com/product/leather-shoulder-bag-16992606959mMnL

Test Result: https://pagespeed.web.dev/analysis/https-fejmo-com-product-leather-shoulder-bag-16992606959mMnL/xcbnszkj60?form_factor=mobile

Please anyone help me diagnose/fix this issue? Thanks in advance.

**Comment by ananuness** [comment]
Hii! You can add the `sharp` package. The problem comes from the default image processor that Next uses. As you can see [here](https://nextjs.org/docs/messages/sharp-missing-in-production) in a production environment (and when your issue appears) it's recommended use `sharp`.

```bash
npm i sharp
# or
yarn add sharp
# or 
pnpm i sharp
```
> If sharp is already installed but can't be resolved you can manually pass the path to it via the NEXT_SHARP_PATH environment variable e.g. NEXT_SHARP_PATH=/tmp/node_modules/sharp, for example.

Hope it helps!
